docker/requirements.txt

```diff
@@ -53,13 +53,13 @@ urbanaccess
 
 # jupyter things
 jupyter==1.0.0
-jupyterlab==1.0.9
+jupyterlab==1.2.3
 jupyterhub==0.9.6
 ipyleaflet==0.11.1
 jupyter-server-proxy==1.1.0
 dask-labextension==1.0.3
 nbdime==1.1.0
-jupyterlab-code-formatter==0.5.0
+jupyterlab_code_formatter==0.5.0
```

Comment:
not sure about this change the python package is at https://pypi.org/project/jupyterlab-code-formatter/ (without underscores)

Reply:
let me test with the `-`

Reply:
it doesn't matter for python package names, so just revert that change, should work